components/portal/src/components/image/version/VersionDeleteDialog.js

```diff
@@ -44,11 +44,11 @@ class VersionDeleteDialog extends React.Component {
 
     handleDeleteVersion = () => {
         const self = this;
-        const {globalState, image, version} = self.props;
-        NotificationUtils.showLoadingOverlay(`Deleting image ${image}`, globalState);
+        const {globalState, org, image, version} = self.props;
+        NotificationUtils.showLoadingOverlay(`Deleting image ${org}/${image}`, globalState);
         HttpUtils.callHubAPI(
             {
-                url: `/images/${image}/${version}`,
+                url: `/images/${org}/${image}/${version}`,
```

Comment:
Shouldn't this URL point to /artifacts/${org}/${image}/${version}

Reply:
fixed